frontend/src/components/widgets/Button.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme) => ({
+  button: {
+    border: "1px solid grey",
```

Comment:
Can you also please style the button to not always turn its children to upper case.